saturn/app/Sharp/CompanyDashboard.php

```diff
@@ -78,6 +79,20 @@ function buildWidgetsData(DashboardQueryParams $params)
                 ->setLabel("Capacities 2")
                 ->setColor("#6b4c9a")
         );
+
+        $this->setPanelData(
+            "activeSpaceships", ["count" => $spaceships->where("state", "active")->first()->count]
+        )->setPanelData(
+            "inactiveSpaceships", ["count" => $spaceships->where("state", "inactive")->first()->count]
+        );
+
+        $this->setListData(
```

Comment:
I'm not sure this widget should be called a "List", since it's really a standard name and it can be confused with entity list, form field list... Maybe something more specific, like "ChartList"? Or something else?

Reply:
I chose ListGroup, "à la" bootstrap